lib/player.rb

```diff
@@ -0,0 +1,67 @@
+require_relative 'scrabble'
+
+module Scrabble
+  class Player
+    attr_reader :name, :plays, :won
+    attr_accessor :tiles
+
+    def initialize(name)
+      @name = name
+      @plays = []
+      @won = false
+      @total_score = 0
+      @tiles = []
+    end
+
+    def play(word)
+      if @won
+        return false
+      end
+      plays << word
+      total_score
+      return Scrabble::Scoring.score(word) # new instance of score
+    end
+
+    def total_score
+      sum_of_scores = []
+      @plays.each do |word|
+        sum_of_scores << Scrabble::Scoring.score(word)
+      end
+      @total_score = sum_of_scores.inject(:+)
```

Comment:
You can probably do this with:  

`      @total_score = @plays.map{ |word| Scrabble::Scoring.score(word)}.inject(:+)`